raiden/api/v1/encoding.py

```diff
@@ -78,6 +80,20 @@ def _deserialize(self, value, attr, data, **kwargs):  # pylint: disable=unused-a
         return value
 
 
+class TimeStampField(fields.DateTime):
```

Comment:
Why can't you use `DateTime` directly? AFAICT it does the same thing.

Reply:
The issue is with sqlite DB API and the paste with errors from the test runner above show what happens. The timestamp fields are represented as strings on sqlite, but python's dbapi will try to turn them into datetime objects implicitly. Given that sqlite does not place the "T" as the separator between the date and time sections of the timestamp, the parsing fails. See the error from the test a few comments above.

This custom field bypasses the dbapi implicit conversion by treating the sqlite field as a simple string when serializing/deserializing, and doing the parsing ourselves.

Reply:
That's not my point. For me it seems that we don't need a custom field. If I look at https://marshmallow.readthedocs.io/en/3.0/_modules/marshmallow/fields.html#DateTime I see that it handles `None` values as your implementation does, and also uses `iso` as the default format.

Reply:
I am trying to explain what happened when I *did use DateTime directly*.

 1. The deserialization part failed and I all got was some stacktrace showing that the part of the code that was throwing the exception was at sqlite dbapi.
 1. The serialilzation part did not throw any errors, but the rendered timestamp had the timezone modifier.

GIven these points, I don't see how we could use the DateTime field directly.